aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -277,7 +277,7 @@ module aptos_framework::aptos_governance {
     }
 
     /// Return a signer for making changes to 0x1 as part of on-chain governance proposal process.
-    public fun get_framework_signer(_proposal: &GovernanceProposal): signer acquires GovernanceResponsbility {
+    public fun get_framework_signer(_proposal: GovernanceProposal): signer acquires GovernanceResponsbility {
```

Comment:
This change is so that get_framework_signer fully consumes the GovernanceProposal resource.